docs/Pflichtenheft.tex

```diff
@@ -290,14 +289,14 @@ \subsubsection{GUI Darstellung}
   \item[FA550] Es gibt eine zweistufige Ampel für den allgemeinen Zustand der \gls{Uberwachungskonsole}.
   \item[FA560] Ist die Ampel rot, wurde mindestens ein Alarm ausgelöst.
   \item[FA570] Ist die Ampel grün, wurde kein Alarm ausgelöst.
-  \item[FA590] Es gibt ein "`Über"'-Fenster, in dem der Programmname, die Version, das Programmsymbol, die Lizenz und die Lizenzen verwendeter Projekte angezeigt werden.
-  \item[FA600] Es gibt ein Einstellungsfenster, in dem alle Einstellungen in einem Reiter "`Allgemeine Einstellungen"' vorgenommen werden können.
-  \item[FA610] In den "Allgemeinen Einstellungen" werden die IP-Adresse und Ports der \glslink{OPC UA Client}{OPC UA Clients} und \glspl{OPC UA Server} angezeigt.
+  \item[FA590] Der Menüeintrag "`Über'" öffnet ein neues Fenster, in dem der Programmname, die Version, das Programmsymbol und die Lizenz angezeigt werden.
+  \item[FA600] Der Menüeintrag "`Einstellungen"' öffnet ein separates Fenster.
+  \item[FA610] In den Einstellungen werden die \glslink{TCP/IP}{IP-Adresse} der \glslink{OPC UA Client}{OPC UA Clients} und der \gls{OPC UA Server}, sowie deren Ports angezeigt.
```

Comment:
Das Komma nach \gls{OPC UA Server} ist falsch. Außerdem: Wenn du hier die Einteilung der Reiter entfernst, musst du es auch noch in den optionalen Funktionalitäten machen oder ich mache es später.